packages/blade/jest.web.config.js

```diff
@@ -24,6 +24,9 @@ const baseConfig = {
     '^\\~utils/(.*)': '<rootDir>/src/utils/$1',
     '^\\~tokens/(.*)': '<rootDir>/src/tokens/$1',
   },
+  globals: {
+    __DEV__: true,
```

Comment:
Won't our consumer need to also add this in their config? What happens if this is missing? Does it simply ignore the error or throws error that __DEV__ not found?